gamemodes/cinema/gamemode/modules/scoreboard/cl_playerlist.lua

```diff
@@ -260,6 +264,10 @@ function PLAYER:SetPlayer( ply )
 	self.Avatar:SetPlayer( ply, 64 )
 	self.Avatar:SetVisible( true )
 
+	if ply:GetFriendStatus() == "friend" then
+		self.FriendIcon:SetImage("chev/icon/friend.png")
```

Comment:
Can you use https://wiki.garrysmod.com/page/Panel/Hide and the opposite function to control the visibility? Having an undefined image panel seems like it might turn into a missing texture or something, it's undefined behavior, and the icon won't dissapear if someone is unfriended.

Reply:
Done